content/experiments/qiskit-blocks.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Qiskit Blocks
+author: James Weaver
+description: The QiskitBlocks application teaches quantum computing concepts using Qiskit, in the context of a Minetest block world.
+image: https://github.com/JavaFXpert/QiskitBlocks/raw/master/quantum_circuits_garden.png
+to: /experiments/quantum-toy-piano
```

Comment:
Can you change this to `/experiments/qiskit-blocks`?

Reply:
Done